easybuild/easyconfigs/r/R-bundle/Ignore-BI_COMM_GLOBAL-on-intel.patch

```diff
@@ -0,0 +1,43 @@
+--- pbdBASE.orig/src/export_blacs/pkg_tools.c   2015-06-02 15:25:57.705288000 +0200
++++ pbdBASE/src/export_blacs/pkg_tools.c        2015-06-02 15:25:29.470539000 +0200
+@@ -25,7 +25,9 @@
+        BLACS_APTS.BI_AuxBuff = &BI_AuxBuff;
+        BLACS_APTS.BI_MyContxts = BI_MyContxts;
+        BLACS_APTS.BI_SysContxts = BI_SysContxts;
++#ifndef __INTEL_COMPILER
```

Comment:
hmm, this is actually dependent on Intel MKL, rather than the Intel compilers, right?

In the `iomkl` module, we have Intel compilers + OpenBLAS + ScaLAPACK + BLACS, there this would be fine?

Maybe there's a better `ifndef`?